File: programmers/약수의 합.js
function solution(n) {
    let sum = 0;
    
    for(let i = 1; i <= Math.sqrt(n); i += 1) {
        if(n % i === 0) {
            if(n / i === i) {
                sum += i;
                
                continue;
            }
            
            sum += i;
            sum += n / i;
        }
    }
    
    return sum
}

File: programmers/약수의 합.java
public class Solution {
    public static int solution(int n) {
        int sum = 0;
        
        for(int i = 1; i <= Math.sqrt(n); i++) {
            if(n % i == 0) {
                if(n / i == i) {
                    sum += i;
                } else {
                    sum += i;
                    sum += n / i;
                }
            }
        }
        
        return sum;
    }
}